pyproject.toml

```diff
@@ -105,7 +105,7 @@ ya-market = "0.1.0"
 [tool.poe.tasks]
 test = "pytest --cov=yapapi --ignore tests/goth"
 goth-assets = "python -m goth create-assets tests/goth/assets"
-goth-tests = "pytest -svx tests/goth"
+goth-tests = "pytest -svx tests/goth --config-override docker-compose.build-environment.release-tag=0.6."
```

Comment:
Why is this parameter used?

Reply:
It's used to enforce using `yagna` release `0.6.x` for the tests.